src/components/Menu/MenuMobile.tsx

```diff
@@ -61,11 +61,7 @@ export const MenuMobile = ({
           cardsLayout={cardsLayout}
           hasLabel={!!label}
           sx={styles}
-          component={
-            isOpenSubMenu && openSubMenu !== MenuKeysEnum.WalletSelect
-              ? 'div'
-              : 'ul'
-          }
+          component={'div'}
```

Comment:
why did we have that before @dennyscode you know?

Reply:
Great comment, thanks @tcheee!

I had added this ternary operator to provide proper markup when necessary. Let´s compare!

This is the current state now without ternary:
![image](https://github.com/user-attachments/assets/23362786-c7ba-4c35-92bd-16d33af3f3db)

This is the menu as it was before:
![image](https://github.com/user-attachments/assets/bb517376-cd8b-4e62-b1f9-b2ff7079013e)

@chybisov @tcheee Wdyt? Knowing this extra logic is annoying, but in regards of markup?

Reply:
@dennyscode, can we always use `ul` here?

Reply:
No, currently we use the Menu in two different ways: One is looping through an array if items (as in MainMenu). In other parts such as the WalletMenu, we do not use such MenuList. To re-use the same "Popper" logic, I have squeezed both worlds into one logic.

Check this:
![image](https://github.com/user-attachments/assets/bd3d21b9-2aa8-4f38-a537-50b0af688ca4)